src/lib/bedrockYaml/bedrockYaml.test.ts

```diff
@@ -59,6 +59,8 @@ describe("Adding a new service to a Bedrock file", () => {
     const k8sBackend = "mybackendservice";
     const pathPrefix = "ingressprefix";
     const pathPrefixMajorVersion = "v2";
+    const BuildVariables = ["foo"];
```

Comment:
It looks like you are expecting upper case, is this expected to be a negative test?

Reply:
For now, we _recommend_ users have variables formatted: Uppercase, Numbers, and Underscores, but this is not _mandatory_. So the logs will output a warning when variables are not in this format.

Reply:
I can uppercase the test variable for consistency.